### DISCUSSION TITLE
Showing organizations on profile should be managed from user profile

### DISCUSSION THREAD

**Discussion by jsoref** [community/community#16560] [Other Feature Feedback, Questions, & Ideas]
The steps to show a user's organizations on the user's profile are incredibly convoluted (the only way to figure out how to do it is to google to find the [documentation](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-your-membership-in-organizations/publicizing-or-hiding-organization-membership) which has 5 steps).

The proper steps should be:
1. Click your profile icon
2. Click your organizations
3. If necessary (and even possible, not sure, I'm not in 50 orgs...), filter to find the organization whose visibility you want to adjust
4. Use the drop down for the row tied to the relevant organization to toggle between Public/Private
5. If you want to change more than one org, repeat step 4.

**Comment by ghost** [comment]
@jsoref Hi! How are you? 
1. So... I have an idea equal to your idea... the difference is that I created a graphical interface. - reference: https://github.com/github/feedback/discussions/9452
2. Hope to help in some way 
3. Another difference I thought would be having a filter to know which organizations are private and which are public.
